Automattic/wp-calypso
# Marketing: Add logos, fix image alignment and button sizes

#### Changes proposed in this Pull Request

- Use brand logos instead of illustrations
- Increase size of illustration in SEO card
- Align images to top of card
- Use `compact` button for secondary CTAs

**Before:**
![Screenshot 2019-05-07 15 56 09](https://user-images.githubusercontent.com/4924246/57338124-17c13000-70e1-11e9-9c7d-8b00d4c7ebc3.png)

**After:**
![Screenshot 2019-05-09 13 54 09](https://user-images.githubusercontent.com/4924246/57485991-f20f6480-7261-11e9-95cc-e8a8f111fde6.png)

#### Testing instructions



1. Go to `http://calypso.localhost:3000/marketing/tools/[site]`
2. Double check if updates mentioned above are reflected
